src/DotnetRelease/ReleaseReportRecords.cs

```diff
@@ -4,4 +4,6 @@ public record ReportOverview(DateTime Timestamp, string Version, IList<ReportFam
 
 public record ReportFamily(string Name, IList<ReportDistribution> Distributions);
 
-public record ReportDistribution(string Name, IList<string> ActiveReleases, IList<string> ActiveReleasesUnsupported, IList<string> ActiveReleasesEOLSoon, IList<string> NotActiveReleasesSupported, IList<string> NotActiveReleasesUnsupported, IList<string> ReleasesMissing);
+public record ReportDistribution(string Name, IList<string> ActiveReleases, IList<string> ActiveReleasesUnsupported, IList<string> ActiveReleasesEOLSoon, IList<string> NotActiveReleasesSupported, IList<string> NotActiveReleasesUnsupported, IList<string> ReleasesMissing, IList<ReportException> Exceptions);
```

Comment:
> `IList<ReportException> Exceptions`

Could we make this an `IList<ExceptionalVersion>` ? We can re-use the types from the other project and then we can simplify the code (no `foreach`).

Reply:
Addressed it :)